src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -566,7 +566,7 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
         }
         
         adjustQuestionNumbers(oldQuestionNumber, newQuestionNumber, questions);
-        updateFeedbackQuestion(newQuestion);
+        updateFeedbackQuestionWithResponseRateCheck(newQuestion);
```

Comment:
The old `updateFeedbackQuestion` is used in `adjustQuestionNumbers`, `shiftQuestionNumbersDown`(only the question numbers are changed) and  `FeedbackQuestionLogicTest` (can be changed to `updateFeedbackQuestionWithResponseRateCheck`). If the old `updateFeedbackQuestion` is removed, the response rate will be checked again and again (no changes made) when adjusting the question numbers.